src/sentry/static/sentry/app/stores/sentryAppStore.jsx

```diff
@@ -11,16 +12,26 @@ const SentryAppStore = Reflux.createStore({
 
   load(items) {
     this.items = items;
+    this.deDup();
     this.trigger(items);
```

Comment:
does this matter that `trigger` is being called with the `items` that are passed in and not the deduped ones?

Reply:
Yup, good catch. Thanks!